packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -157,20 +157,17 @@ type RadioGroupItemImplPrimitive = Polymorphic.ForwardRefComponent<
 >;
 
 const RadioGroupItemImpl = React.forwardRef((props, forwardedRef) => {
-  const { disabled, required, ...itemProps } = props;
   const context = useRadioGroupContext(ITEM_NAME);
   const isChecked = context.value === props.value;
-  const handleChange = composeEventHandlers(itemProps.onCheckedChange, context.onValueChange);
 
   return (
     <Radio
-      {...itemProps}
+      required={context.required}
+      checked={isChecked}
+      {...props}
       name={context.name}
```

Comment:
I'm guessing you left this one here because the prop it omitted from types?

Reply:
nope, missed! 👀